src/aspire/basis/fb_2d.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger(__name__)
 
 
-class FBBasis2D(SteerableBasis2D, FBBasisMixin):
+class FBBasis2D(FBBasisMixin, SteerableBasis2D):
```

Comment:
Could you give me a brief explanation of what's going on with this?

Reply:
Added to main PR thread